### DISCUSSION TITLE
Subber's claw for Builders

### DISCUSSION THREAD

**Discussion by melex750** [colobot/colobot#1514] [Ideas]
![builder1](https://user-images.githubusercontent.com/15256309/159121248-189f6840-a078-4128-8603-6e3687acca56.png)
![builder2](https://user-images.githubusercontent.com/15256309/159121260-74ed60b7-f035-44a5-8bdc-7db8b4217af9.png)

**Comment by tomangelo2** [comment]
I like it. When we were discussing BuilderBots, I thought about making Grabbers able to build with neutron guns in their hood (so it would still work the same as before BuilderBots implementation, just with new animation of opening the hood), to still make it possible to make automated levels with only one robot. This idea looks more native, as it uses existing mechanism of Subber claws instead of carving out animation in current models.

However, there is left to discuss whether should it be default, or switchable with scene parameter.
